quarkus/config-api/src/main/java/org/keycloak/config/StorageOptions.java

```diff
@@ -324,6 +326,15 @@ public String getProvider() {
             .hidden()
             .build();
 
+    public static final Option<String> STORAGE_JPA_DB = new OptionBuilder<>("storage-jpa-db", String.class)
+            .category(OptionCategory.STORAGE)
+            .defaultValue(Database.Vendor.POSTGRES.toString())
+            .expectedValues(Database::getAvailableMapStoreAliases)
+            .description("The database vendor for jpa map storage.")
+//            .hidden() // todo ??
```

Comment:
The decision to mark it as a `hidden` option depends on how you expect users to switch databases. If you are not sure about making it fully supported, marking it as `hidden` makes sense.